lib/pluginmanager/templates/codec-plugin/Gemfile.erb

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+gemspec
+
+logstash_path = ENV['LOGSTASH_PATH'] || '<%= logstash_path %>'
```

Comment:
I've a concern about this.
For other plugins (eg beats input) it's `../../logstash` and the plugin build works fine with it.
Using absolute LS path of a local installation probably is not good, because it's heavily dependent on the a local path. (the same applies for the other plugin templates)

Reply:
Since the `bin/logstash-plugin generate` command takes a `--path` param, the user has full control over where this file gets written. There might not be a simple relative path.

My thought was that this is just a way to make sure the generated code works immediately, but definitely is not "production worthy". Shortly after the user generates this, I'd hope that they'd switch to using the `LOGSTASH_PATH` env var.

Another option could be to remove the `--path` param so that you can deterministically control where the generated code is written? But I expect that would not be acceptable as a backwards-incompatible change. 

Do you have other ideas for how this could be solved?